### PR TITLE
Add code-oriented questions in the first Python workout

### PR DESCRIPTION
**Pull request by nem035**

### CODE REVIEW

File: python/python-core/meet-python/what-is-python.md [hunk 1]
```diff
@@ -47,23 +47,34 @@ You can use it to build websites, web apps and desktop apps. But it's also used
 
 ## Practice
 
-Python is closer to ??? language and further away from ??? language.
+Let's write some Python code! Did you remember how to print a message?
```
Comment (nem035) [inline python/python-core/meet-python/what-is-python.md:50]:
```suggestion
Let's write some Python code! Do you remember how to print a message?
```

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

File: python/python-core/meet-python/what-is-python.md [hunk 2]
```diff
@@ -47,23 +47,34 @@ You can use it to build websites, web apps and desktop apps. But it's also used
...
+- log
 
 ---
 
 ## Revision
 
-Python is both a ??? and general-purpose programming language.
+Python is a programming language that can be used for just about anything.
+
+We call these kinds of languages ??? languages.
```
Comment (nem035) [inline python/python-core/meet-python/what-is-python.md:73]:
```suggestion
We call these kinds of languages ???.
```

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

File: python/python-core/meet-python/why-learn-python.md [hunk 3]
```diff
@@ -55,24 +53,42 @@ Check out the *Learn More link* to dig into more reasons to learn Python.
...
+if answer == "Yes":
+  print("That's the spirit! 🎉") 
+else:
+  print("Don't give up, you're doing great! 🤗")
+```
 
-- less
-- more
-- about the same
+> There's no wrong answers here. The goal is to get you comfortable with reading code :)
```
Comment (nem035) [inline python/python-core/meet-python/why-learn-python.md:68]:
```suggestion
> There're no wrong answers here. The goal is to get you comfortable with reading code :)
```

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

File: python/python-core/meet-python/what-is-python.md [hunk 4]
```diff
@@ -47,23 +47,34 @@ You can use it to build websites, web apps and desktop apps. But it's also used
...
+- say
+- yell
+- log
 
 ---
 
 ## Revision
 
-Python is both a ??? and general-purpose programming language.
+Python is a programming language that can be used for just about anything.
```
Comment (kapnobatai137) [inline python/python-core/meet-python/what-is-python.md:71]: let's switch to the active voice here
```suggestion
Python is a programming language that you can use for just about anything.
```

File: python/python-core/meet-python/what-is-python.md [hunk 5]
```diff
@@ -72,5 +83,8 @@ Python is both a ??? and general-purpose programming language.
 ## Footnotes
 
 [1:Syntax]
-The syntax is the set of rules that define how a program will be written and interpreted. It covers the symbols, indentation and keywords of the language. All programming languages use different syntax.
- 
+The syntax of a programming language sets the rules on how a language can be written and understood by the computer.
+
+The same idea exists in human languages. 
+
+Just like how English has special types of words like *verb* and *nouns*, as well as rules on which should come first to make a valid sentence, Python has a different set of rules on how to make valid Python statements.
```
Comment (kapnobatai137) [inline python/python-core/meet-python/what-is-python.md:90]: we should remove the "Just like" at the start of the sentence. how does this sound?
```suggestion
The English language has unique types of words like *verb* and *nouns* and rules on which should come first to make a correct sentence; Python has a different set of rules on making valid Python statements.
```

File: python/python-core/meet-python/why-learn-python.md [hunk 6]
```diff
@@ -34,9 +34,7 @@ Python is seen as the number one language to learn. Here are a few reasons why:
 
 This makes it an *ideal language for beginners to learn*. And it acts as a stepping-stone for learning other object-oriented [1] languages.
 
-Python is also concise.
-
-For the same task, Python requires *3-5* times less code than Java, and up to *10* times less code than C++.
+Python is also concise and usually requires less code for the same task than languages such as Java or C++.
```
Comment (kapnobatai137) [inline python/python-core/meet-python/why-learn-python.md:37]: let's replace "such as" with "like". I've also added "other" here to keep the same message
```suggestion
Python is also concise and usually requires less code for the same task than other languages like Java or C++.
```

File: python/python-core/meet-python/why-learn-python.md [hunk 7]
```diff
@@ -55,24 +53,42 @@ Check out the *Learn More link* to dig into more reasons to learn Python.
 
 ## Practice
 
-It typically requires ??? code to write the same task in Python than in C++.
+Python is known as an easy-to-read language due to its close similarity to English.
```
Comment (kapnobatai137) [inline python/python-core/meet-python/why-learn-python.md:56]: I think it's safe to remove "known as" here
```suggestion
Python is an easy-to-read language due to its close similarity to English.
```

**Review by kapnobatai137 — CHANGES_REQUESTED** [review]
looks good :+1: I've left some copy improvement suggestions

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/python/python-core/meet-python/python-hello-world.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/meet-python/python-hello-world.md?ref=509ec0eb56065db054b1d6d68810fe6ab2fa6746)
- [ ] [enkidevs/curriculum/python/python-core/meet-python/python-versions.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/meet-python/python-versions.md?ref=509ec0eb56065db054b1d6d68810fe6ab2fa6746)
- [ ] [enkidevs/curriculum/python/python-core/meet-python/what-is-python.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/meet-python/what-is-python.md?ref=509ec0eb56065db054b1d6d68810fe6ab2fa6746)
- [ ] [enkidevs/curriculum/python/python-core/meet-python/why-learn-python.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/meet-python/why-learn-python.md?ref=509ec0eb56065db054b1d6d68810fe6ab2fa6746)

**Review by kapnobatai137 — DISMISSED** [review]

**Review by kapnobatai137 — APPROVED** [review]

**Pull request merged**